gateway/config.go

```diff
@@ -54,5 +74,18 @@ func (cfg *Config) Validate() error {
 	if !strings.HasPrefix(cfg.AlertManagerAddress, "http") {
 		return fmt.Errorf("alertmanager address must start with a valid scheme (http/https). Given is '%v'", cfg.AlertManagerAddress)
 	}
+
+	if cfg.JwtSecret == "" && cfg.JwksURL == "" {
+		return fmt.Errorf("you must set -gateway.auth.jwt-secret and/or -gateway.auth.jwks-url")
```

Comment:
To handle the case when gateway is running with auth disabled and a tenantName is provided
```suggestion
	if cfg.JwtSecret == "" && cfg.JwksURL == "" && cfg.TenantName == "" {
	return fmt.Errorf("you must set -gateway.auth.jwt-secret and/or -gateway.auth.jwks-url" or -gateway.auth.tenantName)
```